lib/twilio/device.ts

```diff
@@ -2065,11 +2046,11 @@ namespace Device {
     /**
      * Overrides the native RTCPeerConnection class.
      *
-     * By default, the SDK will use the `unified-plan` SDP format if the browser supports it.
+     * By default, the SDK will only use the `unified-plan` SDP format.
      * Unexpected behavior may happen if the `RTCPeerConnection` parameter uses an SDP format
      * that is different than what the SDK uses.
      *
-     * For example, if the browser supports `unified-plan` and the `RTCPeerConnection`
+     * For example, if the browser only supports `unified-plan` and the `RTCPeerConnection`
      * parameter uses `plan-b` by default, the SDK will use `unified-plan`
```

Comment:
There's a few references to `plan-b` in this doc. I'm not too in the loop but it appears that `plan-b` is just gone entirely right? It was Chromium specific and now it's gone with the latest Chromium updates yeah? Should we remove all references to it?

Reply:
Yes, plan-b is gone entirely on the latest versions of all major browsers. However, we cannot remove plan-b references in this specific section. For example, VDI providers like Citrix still supports plan-b. This doc basically just tells you, if you're going to override RTCPeerConnection, make sure you explicitly say unified plan especially if your VDI provider is defaulting to plan-b.